sdk/eventhub/Azure.Messaging.EventHubs.Processor/perf/Scenarios/ProcessorReceiveTest.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Processor.Perf.Infrastructure;
+
+namespace Azure.Messaging.EventHubs.Processor.Perf.Scenarios
+{
+    public class ProcessorReceiveTest : ProcessorTest<ProcessorOptions>
+    {
+        public ProcessorReceiveTest(ProcessorOptions options) : base(options)
+        {
+            EventProcessorClient.ProcessEventAsync += ProcessEventAsync;
+            EventProcessorClient.ProcessErrorAsync += ProcessErrorAsync;
+        }
+
+        private Task ProcessEventAsync(ProcessEventArgs arg)
+        {
+            try
+            {
+                EventRaised();
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                Environment.Exit(1);
```

Comment:
I just wrote this code, but I don't really like `Environment.Exit()` here either, since it prevents graceful cleanup.  Instead, I think I will add an `EventPerfTest.ErrorRaised(Exception)` method to handle this more gracefully.